docs/iris/src/whatsnew/3.0.4.rst

```diff
@@ -22,6 +22,11 @@ This document explains the changes made to Iris for this release
       :meth:`iris.fileformats.pp.PPDataProxy.__getitem__`. The check for empty slicings
       is no longer needed since (:pull:`4135`) was added. (:pull:`4141`)
 
```

Comment:
@pp-mo You still need to remove the above entry, as that was a patch applied in [v3.0.2](https://scitools-iris.readthedocs.io/en/v3.0.3/whatsnew/3.0.2.html)

Reply:
If so, why isn't it already in the whatsnew here under 3.0.2 changes ?

Reply:
It seems to have vanished between the whatsnew/v3.0.3 and v3.0.4 ...